tslint.json

```diff
@@ -1,4 +1,10 @@
-{
```

Comment:
This looks like you are unbalancing the brackets. Where they implanced to begin with or is this a mistake